Library/Homebrew/utils.rb

```diff
@@ -770,3 +770,23 @@ def number_readable(number)
   (numstr.size - 3).step(1, -3) { |i| numstr.insert(i, ",") }
   numstr
 end
+
+def truncate_text_to_approximate_size(s, max_bytes, opts = {})
```

Comment:
Unsure how to read this: by "that' do you mean "the _s suffix is okay, but this usage is not", or "okay, this usage of `s` is fine too"?